_examples/ceph-source-example.adoc

```diff
@@ -60,14 +78,16 @@ No architecture, this is a connectivity example.
 
 2. Start a Jupyter notebook with the commands:
 
-   oc new-app rimolive/notebook
-   oc expose svc/notebook
+  oc new-app rimolive/base-notebook \
+    -e JUPYTER_NOTEBOOK_PASSWORD=supersecret \
+    -e JUPYTER_NOTEBOOK_X_INCLUDE=https://radanalytics.io/assets/ceph-source-example/ceph-source-example.ipynb
+  oc expose svc/base-notebook
 
-3. Find the connection url using the following command:
+3. Open the Jupyter URL created by OpenShift and type the password you set in the previous command.
 
-   oc logs dc/pyspark-hdfs-notebook | grep localhost | sed "s/localhost:8888/$(oc get routes/pyspark-hdfs-notebook --template='{{.spec.host}}')/"
+4. Look for the notebook named `s3-source-example.ipynb`, open it and follow the instructions. Make sure in the line where the notebook connects to the spark cluster you changed with the cluster name you used in step 1 and in the line where you set up the Ceph connection you change with the IP where you Ceph instance is running.
```

Comment:
shouldn't this be `ceph-source-example.ipynb` instead of `s3-source-example.ipynb`?